concourse/scripts/pipelines.sh

```diff
@@ -46,12 +29,16 @@ bosh_instance_profile: ${BOSH_INSTANCE_PROFILE}
 concourse_instance_type: ${CONCOURSE_INSTANCE_TYPE}
 concourse_instance_profile: ${CONCOURSE_INSTANCE_PROFILE}
 enable_datadog: ${ENABLE_DATADOG}
-datadog_api_key: ${datadog_api_key:-}
-datadog_app_key: ${datadog_app_key:-}
+datadog_api_key: ${DATADOG_API_KEY:-}
+datadog_app_key: ${DATADOG_APP_KEY:-}
 concourse_auth_duration: ${CONCOURSE_AUTH_DURATION:-5m}
 EOF
 }
 
+if [ "${ENABLE_DATADOG}" = "true" ] ; then
+  eval "$("${SCRIPT_DIR}"/../../scripts/manage-datadog-secrets.sh retrieve)"
```

Comment:
Copy from discussion offline: this failed when I ran `make dev upload-datadog-secrets DEPLOY_ENV=dcarley` and then `ENABLE_DATADOG=true make dev deployer-concourse bootstrap`. It looks like it's output from aws-cli (version 1.9.2):
```
➜  paas-bootstrap git:(131876783_lose_ends) ENABLE_DATADOG=true make dev deployer-concourse bootstrap
vagrant/deploy.sh
Bringing machine 'default' up with 'aws' provider...
==> default: The machine is already created.
Setting up SSH tunnel to concourse...
Master running (pid=6026)
#!/bin/bash
Downloading fly command...

Doing fly login
username: admin
password:
token saved
Doing fly sync
downloading fly from http://localhost:8080... update successful!
/Users/dcarley/projects/paas/paas-bootstrap/concourse/scripts/pipelines.sh: line 39: download:: command not found
make: *** [bootstrap] Error 127
```